pkgs/applications/editors/emacs/generic.nix

```diff
@@ -58,16 +58,26 @@ let emacs = stdenv.mkDerivation (lib.optionalAttrs nativeComp {
 
   enableParallelBuilding = true;
 
+  patches = patches ++ (
+    lib.optional (stdenv.isDarwin && stdenv.isAarch64)
+      [
+        (fetchpatch {
+          url = "https://github.com/emacs-mirror/emacs/commit/868f51324ac96bc3af49a826e1db443548c9d6cc.patch";
+          sha256 = "sha256-DJcqAJfcg4YCHvuMxumBzlu+SPMQG6+8FcveUTFPN2M=";
+        })
+      ]
+  );
+
   postPatch = lib.concatStringsSep "\n" [
     (lib.optionalString srcRepo ''
       rm -fr .git
     '')
 
     # Reduce closure size by cleaning the environment of the emacs dumper
-    ''
+    (lib.optionalString ((stdenv.isDarwin && stdenv.isAarch64) == false) ''
```

Comment:
I think this is a red herring, I'm going to test if this works without this change...

Reply:
not a red herring, needs to be there

Reply:
I don't see a failure if I remove this. What was happening?

Reply:
double blunder, this really isn't needed, now removed.